kiva/ui
# GROW-82 New MG setup design experiment

Sorry for the giant PR, it might be easier to view the files whole instead of as diffs. 
Overview:

- Renames MonthlyGoodSetupPage.vue to MonthlyGoodSetupPage**Control**.vue 
- Builds the new design from GROW-82 to MonthlyGoodSetupPage**Variant**.vue
- MonthlyGoodSetupPage.vue now checks the uiexperiment and determines which component (control or variant) to show.
- Some tweaks to KvCheckoutSteps to make it work well in this scenario and the checkout.

- To test:
  - add a setting named `uiexp.mg_setup` to your vm admin settings manager with this data:
`{ "name": "MonthlyGoodSetup", "enabled": true, "startTime": "2020-05-01T00:00", "endTime": "2021-05-01T00:00", "control": { "key": "control", "name": "Control" }, "distribution": { "control": 0.5, "shown": 0.5 } }`

  - To see the new design go to:
  /monthlygood/setup?amount=25&category=eco_friendly&setuiab=mg_setup.shown
  or the control:
  /monthlygood/setup?amount=25&category=eco_friendly&setuiab=mg_setup.control

  - You can now go to /monthlygood and choose different amounts and categories. Try the covid fund please.
